### PR TITLE
Add server events log

### PR DESCRIPTION
**Pull request by PawelPodkalicki**

### CODE REVIEW

File: app/Console/Commands/AdsFetchHosts.php [hunk 1]
```diff
@@ -84,6 +86,7 @@ public function handle(AdsClient $adsClient): void
         $this->comment('Cleaning up old hosts...');
         $removed = $this->removeOldHosts();
         $marked = $this->markHostsWhichDoesNotBroadcast();
+        ServerEvent::dispatch(ServerEventType::HostBroadcastProcessed, ['marked' => $marked, 'removed' => $removed]);
```
Comment (m-pilarczyk) [inline app/Console/Commands/AdsFetchHosts.php]: Add registered hosts counter

**Review by m-pilarczyk — CHANGES_REQUESTED** [review]

File: app/Console/Commands/AdsFetchHosts.php (hunk 1 above)
Comment (PawelPodkalicki) [reply app/Console/Commands/AdsFetchHosts.php]: Added

File: app/Console/Commands/AdsFetchHosts.php [hunk 2]
```diff
@@ -106,14 +117,18 @@ private function getTimeOfFirstBlock(int $timeNow): int
...
         try {
-            $resp = $adsClient->getBroadcast($blockId);
-            $broadcastArray = $resp->getBroadcast();
+            $response = $adsClient->getBroadcast($blockId);
+            $broadcastArray = $response->getBroadcast();
 
             foreach ($broadcastArray as $broadcast) {
-                $this->handleBroadcast($broadcast);
+                $wasHostFound = $this->handleBroadcast($broadcast);
+                if ($wasHostFound) {
```
Comment (m-pilarczyk) [inline app/Console/Commands/AdsFetchHosts.php]:
```suggestion
                if ($this->handleBroadcast($broadcast)) {
```

File: app/Console/Commands/AdsFetchHosts.php [hunk 3]
```diff
@@ -106,14 +117,18 @@ private function getTimeOfFirstBlock(int $timeNow): int
...
+        $foundHosts = 0;
         try {
-            $resp = $adsClient->getBroadcast($blockId);
-            $broadcastArray = $resp->getBroadcast();
+            $response = $adsClient->getBroadcast($blockId);
+            $broadcastArray = $response->getBroadcast();
 
             foreach ($broadcastArray as $broadcast) {
-                $this->handleBroadcast($broadcast);
+                $wasHostFound = $this->handleBroadcast($broadcast);
```
Comment (m-pilarczyk) [inline app/Console/Commands/AdsFetchHosts.php]:
```suggestion
```

**Review by m-pilarczyk — CHANGES_REQUESTED** [review]

**Review by m-pilarczyk — APPROVED** [review]

**Pull request merged**